demisto_sdk/commands/common/tools.py

```diff
@@ -792,11 +792,16 @@ def _write():
 
 
 @lru_cache
-def get_file(file_path: Union[str, Path], type_of_file: str, clear_cache: bool = False):
+def get_file(
+    file_path: Union[str, Path],
+    type_of_file: Optional[str] = None,
+    clear_cache: bool = False,
+):
     if clear_cache:
         get_file.cache_clear()
-
     file_path = Path(file_path)  # type: ignore[arg-type]
+    if not type_of_file:
+        type_of_file = file_path.suffix
```

Comment:
Awesome 🔥 

Since we add functionality here (deducting the `file_type`), please add a debug log before `return {}` at the end, saying we couldn't detect the file type.